website/docs/jellyfish/api/loan.md

```diff
@@ -277,4 +277,32 @@ interface UTXO {
   txid: string
   vout: number
 }
+```
+
+## getVault
+
+Returns information about vault.
+
+```ts title="client.loan.createVault()"
+interface loan {
+  getVault (vaultId: string): Promise<VaultDetails>
+}
+
+interface VaultDetails {
+  loanSchemeId: string
+  ownerAddress: string
+  isUnderLiquidation: boolean
+  batches?: AuctionBatchDetails[]
+  collateralAmounts?: string[]
+  loanAmount?: string[]
+  collateralValue?: BigNumber
+  loanValue?: BigNumber
+  currentRatio?: number
+}
+
+interface AuctionBatchDetails {
+  index: number
```

Comment:
Should be BigNumber

Reply:
index is a Whole number. so we are safe.

Reply:
https://github.com/DeFiCh/jellyfish/pull/569/files#r713796485